### ISSUE TITLE
Problem with Dataverse Collection publish and edit button

### ISSUE THREAD

**Issue by IlariaBelvedere** [IQSS/dataverse-docker#71]
Hello everyone,

when I go the Dataverse sub Collections (not the root), the publish and edit buttons seem to be disabled and so I am not able to publish or edit them.
What can I do?
Thank you for your assistance

**Comment by IlariaBelvedere** [comment]
As you can see in the first screen (sorry for the low quality of the photos), I was able tu publish the Dataverse Collection called “Dipartimento di Filologia, Letteratura e Linguistica Sample” via API, so the publish button isn’t there anymore but there’s still the edit button and I can’t click on it, as it happened before with publish button. In the second screen, another collection which is unpublished (with the two buttons disabled).

Da: Vyacheslav ***@***.***>
Inviato: giovedì 14 luglio 2022 22:29
A: ***@***.***>
Cc: Ilaria ***@***.***>; ***@***.***>
Oggetto: Re: [IQSS/dataverse-docker] Problem with Dataverse Collection publish and edit button (Issue #71)


Can you please attach screenshot?

**Comment by 4tikhonov** [comment]
Sorry, screenshots aren't attached.

**Comment by IlariaBelvedere** [comment]
Sorry, here they are:


![screen 1](https://user-images.githubusercontent.com/82025109/179089967-3731ff93-7ee2-49b8-b6a8-0b7590dbbe09.jpg)

![screen 2](https://user-images.githubusercontent.com/82025109/179090028-2c18ce8a-5c98-4c59-80b2-c875a6ccfe56.jpg)

**Comment by 4tikhonov** [comment]
Right, I think you should publish Root subdataverse first to get other subdataverses published.

**Comment by IlariaBelvedere** [comment]
Yes, I did publish the root one with the publish button, but After that the others had still their bottons disabled.

Sent from Android device

Il 14 lug 2022 23:40, Vyacheslav Tykhonov ***@***.***> ha scritto:

Right, I think you should publish Root subdataverse first to get other subdataverses published.

**Comment by IlariaBelvedere** [comment]
So I published the one under the root "Dipartimento ecc." with the API. I am still not able to edit It via interface.

Sent from Android device

Il 14 lug 2022 22:29, Vyacheslav Tykhonov ***@***.***> ha scritto:

Can you please attach screenshot?
